lib/foreman/telemetry.rb

```diff
@@ -96,6 +96,13 @@ def enabled?
       @sinks.count > 1
     end
 
+    def allowed_tags(labels)
```

Comment:
might want to memoize this method, otherwise, you're doing this processing on each metric increment. Not sure it's a huge issue, but it could add up...

Reply:
Oh absolutely this should have been a class variable in fact, an overlook. Thanks, that's actually an ugly regression in fact.

Reply:
Wait it's a singleton, memorizing is good enough.